src/lando/main/models/repo.py

```diff
@@ -46,6 +47,18 @@ class RepoError(Exception):
     pass
 
 
+# Dict of hook names to docstring, suitable as human-friendly choices.
+HOOKS_CHOICES = {chk.__name__: chk.__doc__ or chk.__name__ for chk in ALL_CHECKS}
+
+
+def get_default_hooks() -> list[str]:
+    """Returns a list of all known hook names, suitable as a default value.
+
+    Assuming a normal repository, we enable everything but the BugReferencesCheck, which
+    is only relevant for Try-type repos."""
+    return [hook for hook in HOOKS_CHOICES if hook != BugReferencesCheck]
```

Comment:
It looks like `hook` would be the string name of the hook, and thus `hook != BugReferencesCheck` will be `True` even when `hook == "BugReferencesCheck"`. I think we need to compare to `BugReferencesCheck.__name__`.